ote/src/cljs/ote/ui/validation.cljs

```diff
@@ -17,8 +17,14 @@
 (def phone-number-regex #"^((\+?\d{0,15})|(\d{0,16}))$")
 
 (defn empty-value? [val]
-  (or (nil? val)
-      (str/blank? val)))
+  ;; empty? function call fails for following types.
+  ;; Therefore a specific type check needed.
+  (if (or (boolean? val)
+          (keyword? val))
+    false
+    (or (nil? val)
+        (str/blank? val)
+        (empty? val))))
```

Comment:
Rather than special case the types `empty?` won't work with, specifically check for a collection:
`(and (coll? val) (empty? val))`